docs/en/observability/synthetics-security-encryption.asciidoc

```diff
@@ -0,0 +1,20 @@
+[[synthetics-security-encryption]]
+= Synthetics Encryption and Security
+
+Elastic Synthetics was designed with security in mind encrypting both persisted and transmitted data.
+This page catalogs the points within the Elastic Synthetics app data is either stored or transmitted in an encrypted fashion.
+
+== Synthetics UI (Kibana App)
+
+Data is stored in Kibana Secure Saved Objects[https://www.elastic.co/guide/en/kibana/master/xpack-security-secure-saved-objects.html], 
+with sensitive fields encrypted. These fields include your script source, params, and global params.
```

Comment:
This is even more comprehensive than this, but not sure if we need to specify every field.

Reply:
Yeah, I don't think we need them all